setup/blobfuse2.service

```diff
@@ -14,39 +14,14 @@ Environment=BlobTmp=/home/AzureUser/tmpblobfuse
 # Config file path
 Environment=BlobConfigFile=/home/AzureUser/azure-storage-fuse/blobfuse2.yaml
 
-# Specifies the storage account blobfuse targets.
-Environment=AZURE_STORAGE_ACCOUNT=<account.name>
```

Comment:
These are required, as not all customers provide their credentials in config file.